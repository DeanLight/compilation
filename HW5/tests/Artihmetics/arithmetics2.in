void main()
{
	int a1 = 4/2/2/2*2;
	printi(a1);
	int a2 = 7/2*3/4*5;
	print("\nyou are ");
	printi(a2);
}