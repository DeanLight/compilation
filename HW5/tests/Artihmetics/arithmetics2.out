Loaded: ./exceptions.s
0
you are 10